README.md

```diff
@@ -177,9 +177,7 @@ instructions for other platforms, and just in general, we recommend you see [Ras
 1. Setup a CMake build directory.
       For example, if not using an IDE:
       ```
-      $ mkdir build
-      $ cd build
-      $ cmake ..
+      $ cmake -S . -B build
```

Comment:
IMHO we should either explain what these short command-line switches mean...

Reply:
Something like adding
> `cmake -S <source_dir> -B <build_dir>`

below the command?

Reply:
Yeah, or just something like
> The cmake `-S` flag indicates the source directory, and `-B` tells cmake where to put the output files.

? :shrug: